bin/varnishncsa/varnishncsa_options.h

```diff
@@ -78,6 +78,9 @@
 	VOPT("E", "[-E]", "Show ESI requests",				\
 	    "Show ESI requests, implies client mode."			\
 	)
+#define NCSA_OPT_j                                          \
+	VOPT("j", "[-j]", "Print logs to stdout as JSON", \
+		 "Print logs to stdout as JSON.")
```

Comment:
Pedantic police:

- this isn´t printing as JSON, but rather escaping format components as JSON strings.
- whitespace violations :police_officer:

Reply:
Oops, our mistake. Thanks for catching this.